src/features/projects/server/projectsRouter.ts

```diff
@@ -67,4 +67,45 @@ export const projectsRouter = createTRPCRouter({
 
       return true;
     }),
+
+  transfer: protectedProcedure
+    .input(
+      z.object({
+        projectId: z.string(),
+        email: z.string().email(),
+      })
+    )
+    .mutation(async ({ input, ctx }) => {
+      throwIfNoAccess({
+        session: ctx.session,
+        projectId: input.projectId,
+        scope: "project:transfer",
+      });
+
+      const user = await ctx.prisma.user.findUnique({
```

Comment:
can we have a try .. catch block here where we return a 500 with internal error and log the exception to the console?